app/lib/registries/topic_taxonomy_registry.rb

```diff
@@ -7,7 +7,7 @@ def [](content_id)
     end
 
     def taxonomy_tree
-      Rails.cache.fetch(CACHE_KEY, expires_in: 1.hour) do
+      @taxonomy_tree = Rails.cache.fetch(CACHE_KEY, expires_in: 1.hour) do
         taxonomy_tree_as_hash
       end
     rescue GdsApi::HTTPServerError
```

Comment:
Will the memoization variable be set as the default `{}` if there's an error fetching the registry data? If so, do we want to retry fetching the data if that happens?

Reply:
The default `{}` is outside the memoization block so it should be fine and will be retried

Reply:
Awesome :)